hail/src/main/scala/is/hail/expr/types/physical/PInt32.scala

```diff
@@ -1,6 +1,7 @@
 package is.hail.expr.types.physical
 
 import is.hail.annotations.{Region, UnsafeOrdering, _}
+import is.hail.asm4s
```

Comment:
Not needed I think, you import coerce below

Reply:
fixed across all the primitives